liblangutil/SourceReferenceExtractor.h

```diff
@@ -64,12 +64,12 @@ namespace SourceReferenceExtractor
 	struct Message
 	{
 		SourceReference primary;
-		std::string category; // "Error", "Warning", ...
+		std::string severity; // "Error", "Warning", "Info", ...
```

Comment:
Hmm, shouldn't this use the enum? Not sure what this does here but is suspect.

Reply:
Apparently it's also indirectly used by other places in the compiler, like CommandLineInterface.cpp, that do not adhere to the severity standard:
```
solc/CommandLineInterface.cpp:		formatter.printExceptionInformation(_exception, "Compiler error");
printExceptionInformation(SourceReferenceExtractor::extract(m_charStreamProvider, _exception, _severity));
```

Reply:
@axic can we keep the changes to a min here without diving into other subsystems, and maybe do that after?

Reply:
@cameel what do you think?

Reply:
Yeah, this is just a refactor so I think it would be reasonable to clean it up separately so that we can have the new severity in the upcoming release.

Reply:
> @axic can we keep the changes to a min here without diving into other subsystems, and maybe do that after?

Can you open tracking issue?